packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -279,7 +285,7 @@ export class Span implements api.Span, ReadableSpan {
    * @param value Attribute value
    * @returns truncated attribute value if required, otherwise same value
    */
-  private _truncateToSize(value: SpanAttributeValue): SpanAttributeValue {
```

Comment:
In the SDK, we still keep the deprecated `SpanAttributeValue` because the SDK is required to support the older version of `@opentelemetry/api`.

Would you mind reverting these unrelated changes? Thank you!

Reply:
i can.